wfl/fit/mace.py

```diff
@@ -125,30 +133,29 @@ def fit(fitting_configs, mace_name, mace_fit_params, mace_fit_cmd, ref_property_
     try:
         remote_cwd = os.getcwd()    
 
-        # Copy training set xyz file to run_dir.
-        # If initiated with list of ASE objects then write xyz file in run_dir. 
-        if not fitting_configs.one_file():
-            fitting_configs = list(fitting_configs)
-            _write_fitting_configs(fitting_configs, mace_fit_params, run_dir, fit_cfgs_fname)
-        else:
-            copyfile(fitting_configs.one_file(), f"{run_dir}/{fitting_configs.one_file().stem}.xyz")
-
         # previous checkpoint file should be moved by remote_info.input_files function 
         if prev_checkpoint_file != None:
             checkpoint_dir = Path(run_dir / "checkpoints")
```

Comment:
Having the `Path( .. )` is useless here, because the `/` operator only works on `Path` objects to begin with.  If you're worried that `run_dir` isn't a `Path`, you need `Path(run_dir) / "checkpoints"`, but `run_dir` is used in so many places, you should just do `run_dir = Path(run_dir)` at the beginning of the function, if needed, and then you can assume it's a `Path` later in the function.

Reply:
Please use python recommended `is not None` rather than `!= None`